rust/Dockerfile

```diff
@@ -73,6 +84,7 @@ RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-
 # mold: A Modern Linker
 ENV MOLD_VERSION v1.0.1
 RUN set -e \
+    && cd $HOME \
```

Comment:
NIT: I wonder, if setting WORKDIR explicitly could help is to avoid this repetition

Reply:
Looks like it does